### DISCUSSION TITLE
Don't resume on replug if it's some time ago

### DISCUSSION THREAD

**Discussion by PaulWoitaschek** [PaulWoitaschek/Voice#1310] [Ideas]
Check when the headset was plugged out and only resume upon replug if it's 5 ago or less.

**Comment by versavius** [comment]
Hi Paul sorry to be pendatic but you wrote:

> if it's 5 ago or less

Is this 5 seconds or less ?

If we add this option can we have a option in settings to enable/disable please.

**Comment by jtagcat** [comment]
I feel 5s is too little.
Should it be user configurable? A toggle for the resume on replug / bt connected (seperate toggles if possible?), if toggle is true, show a slider, from 1 to 60, incrementing in 5 (seconds), after 60 is reached, switch to minutes with a step of 15s, until around 10m. Last step is infinite, the default.
